emojirades/tests/FileFixture.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+class FileFixture:
+    """
+    Test helper to read fixtures files with relative path
+    path: pass in the path after `fixtures`
+    """
+    def __init__(self, path):
+        dirname = os.path.dirname(__file__)
+        self.filename = os.path.join(dirname, os.path.join("fixtures", path))
+
+    def open(self):
+        return open(self.filename, 'r')
```

Comment:
I'm a bit of a fan of double quotes, black run maybe?

Reply:
argh.. old habit lol.. oh we have 'black', don't we? haha.. will do that before squashing.